x-pack/plugins/observability_solution/slo/public/embeddable/slo/burn_rate/types.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  SerializedTitles,
+  PublishesWritablePanelTitle,
+  PublishesPanelTitle,
+} from '@kbn/presentation-publishing';
+import { DefaultEmbeddableApi } from '@kbn/embeddable-plugin/public';
+import { Subject } from 'rxjs';
+import {
+  type CoreStart,
+  IUiSettingsClient,
+  ApplicationStart,
+  NotificationsStart,
+} from '@kbn/core/public';
+
+export interface EmbeddableProps {
+  sloId: string;
+  sloInstanceId: string;
+  duration: string;
+  reloadSubject?: Subject<boolean>;
+  onRenderComplete?: () => void;
```

Comment:
@kdelemme This looks like a leftover from the previous framework. `<BurnRate />` doesn't pass onRenderComplete. I see alerts and error budget embeddables also reference onRenderComplete, but it should be removed there as well (in a separate PR). What do you think?

Reply:
Let me check if I can remove these

Reply:
removed in https://github.com/elastic/kibana/pull/189429/commits/2c40a611b351327f98c3f341b657ff8c362c0669